src/state-managers/url-state-manager.js

```diff
@@ -1,24 +1,50 @@
 // @flow
 
+function setSearchParam(key: string, value: string) {
+  const searchParams = new URLSearchParams(
+    new URL(window.location.href).searchParams,
+  );
+  searchParams.set(key, value);
+  return searchParams.toString() || '';
+}
+
+function getSearchParam(key: string) {
+  const { href } = window.location;
+  const searchParams = new URLSearchParams(new URL(href).searchParams);
```

Comment:
No need to double wrap

```suggestion
  const searchParams = new URL(href).searchParams;
```